app/js/lib/foxbox/foxbox.js

```diff
@@ -89,15 +89,15 @@ export default class Foxbox extends Service {
         this.togglePolling(this[p.settings].pollingEnabled);
       });
       this.togglePolling(this[p.settings].pollingEnabled);
-
-      this.recipes = new Recipes({
-        settings: this[p.settings],
-        net: this[p.net]
-      });
     });
 
     return this._initUserSession()
       .then(() => {
+        this.recipes = new Recipes({
+          settings: this[p.settings],
+          net: this[p.net]
```

Comment:
Looks good, the only thing that worries me a bit is that ```net``` is initialized once discovery is resolved.